recipes-wpe/wpeframework/include/firmwarecontrol.inc

```diff
@@ -0,0 +1,38 @@
+FIRMWARECONTROL_PLATFORM ?= "none"
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'rpi', '\
+     -DPLUGIN_FIRMWARECONTROL_PLATFORM_RPI=ON \
+', '', d)}"
+
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'broadcom', '\
+     -DPLUGIN_FIRMWARECONTROL_PLATFORM_BCM=ON \
+', '', d)}"
+
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'arris', '\
+     -DPLUGIN_FIRMWARECONTROL_PLATFORM_BCM=OFF \
+', '', d)}"
+
+# Set dependencies
+WPE_FIRMWARECONTROL_DEP ?= "${@bb.utils.contains('FIRMWARECONTROL_PLATFORM', 'rpi', 'mfrlibs-rpi', '', d)}"
```

Comment:
I would get rid of the WPE_FIRMAWARECONTROL_DEP flag and just use the expresion in the DEPENDS